lib/profiling/lfric/README.md

```diff
@@ -0,0 +1,34 @@
+# PSyclone Wrapper Library for LFRic
+
+This is a wrapper library that maps PSyclone's PSyData profiling API
+to the timer functionality provided in LFRic. Note that the module
+is following the LFRic naming convention, meaning the library
+is called libprofile_psy_data_mod.a.
```

Comment:
You need to say that this isn't just a wrapper library since it actually incorporates the LFRic timer code.

Reply:
I've updated all the docs to (hopefully) properly describe this situation.